libsupport/include/katana/ProgressTracer.h

```diff
@@ -143,10 +143,9 @@ class KATANA_EXPORT ProgressTracer {
   uint32_t GetHostID() { return host_id_; }
   uint32_t GetNumHosts() { return num_hosts_; }
 
-  /// Close is called when a tracer is finished processing spans
-  /// This should be called to ensure any and all buffered spans are
-  /// flushed
-  virtual void Close() = 0;
+  /// Finish is called when SharedMemSys/DistMemSys finish
```

Comment:
It is better to describe what this does without reference to SharedMemSys or DistMemSys.

E.g.,

"""
Finish closes the active span and its parent spans if present and resets the active span to the unnamed root span.
"""